pancancer_evaluation/exceptions.py

```diff
@@ -10,8 +10,7 @@ class NoTrainSamplesError(Exception):
     This allows calling scripts to choose how to handle this case (e.g. to
     print an error message and continue, or to abort execution).
     """
-    def __init__(self, *args):
-        super().__init__(*args)
+    pass
```

Comment:
Just curious, why did you convert the init functions for your exceptions? Is the superclass constructor called on init by default? Or does it not matter because you aren't passing messages or something?

Reply:
Based on [this stackoverflow thread](https://stackoverflow.com/a/1319675), it seems like if all you're doing is passing messages you don't need to call the constructor manually or override anything. I'm not exactly sure how that works internally, but I definitely see the messages when I call my custom exceptions (i.e. the init call didn't seem to be adding anything).